### PR TITLE
Follow-up tasks for MySQL rewrite

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-3077
https://issues.redhat.com/browse/DBZ-3086
https://issues.redhat.com/browse/DBZ-3093

### CODE REVIEW

**Comment by jpechane** [comment]
@gunnarmorling This is ready for review, DBZ-3099 will be handled in a separate PR

**Comment by gunnarmorling** [comment]
@jpechane, can you please add a few sentences what's happening on a high level here? Let's make that a habbit for any PR of this size.

**Comment by gunnarmorling** [comment]
Also link to issue, please. Have some compassion for the pour soul reviewing such large PR ;)

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java [hunk 1]
```diff
@@ -786,47 +702,51 @@ protected void handleUpdate(Event event) throws InterruptedException {
...
                 for (int row = startingRowNumber; row != numRows; ++row) {
-                    count += recordMaker.delete(rows.get(row), ts, row, numRows);
+                    changeEmitter.emit(tableId, rows.get(row));
+                    count++;
                 }
                 if (LOGGER.isDebugEnabled()) {
                     if (startingRowNumber != 0) {
-                        LOGGER.debug("Recorded {} delete record(s) for last {} row(s) in event: {}",
-                                count, numRows - startingRowNumber, event);
+                        LOGGER.debug("Emited {} {} record(s) for last {} row(s) in event: {}",
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java:733]: Emitted

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java [hunk 2]
```diff
@@ -786,47 +702,51 @@ protected void handleUpdate(Event event) throws InterruptedException {
...
+                                count, changeType, numRows - startingRowNumber, event);
                     }
                     else {
-                        LOGGER.debug("Recorded {} delete record(s) for event: {}", count, event);
+                        LOGGER.debug("Emited {} {} record(s) for event: {}", count, changeType, event);
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java:737]: Same as above.

**Review by gunnarmorling — COMMENTED** [review]
Unrelated test failure tracked at https://issues.redhat.com/browse/DBZ-3107

**Comment by gunnarmorling** [comment]
Closing, as it's superseded by #2123.

**Pull request closed without merging**